schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+    id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
[OPTIONAL]
- [ ] You can also mark the id column as primary key,  see the below snippet as a guide
`PRIMARY KEY (id)`